### ISSUE TITLE
Integration Tests

### ISSUE THREAD

**Issue by lukasmartinelli** [mapbox/osm-wayback#24]
@jenningsanderson should we integrate https://github.com/mapbox/osm-wayback/tree/master/test into Travis?
Can we make repeatable tests out of those?

**Comment by jenningsanderson** [comment]
Yes, let's do it, we just need to write them into a framework that validates the output, probably Node or Python to validate the JSON output?
